packages/api-clients/open-api/apiGatewayApi.yml

```diff
@@ -2928,7 +2928,7 @@ components:
           type: string
           pattern: '^[a-zA-Z0-9\s\-_]+$'
           minLength: 2
-          maxLength: 16
+          maxLength: 24
```

Comment:
Making this pattern match the one of the attribute seed, otherwise it fails parsing when returning an object with data that was accepted on creation

<img width="327" alt="image" src="https://github.com/user-attachments/assets/7f1985f2-839b-436e-9a33-6ad4d4ec2153">